homework/git-homework-1.md

```diff
@@ -16,10 +16,19 @@
 ### Task 2
 1. Create a new repository on GitHub
     * It should either be public, or you must invite the teaching team as collaborators so we can see your work.
-1. Add your GitHub repository as a remote for your local `gitassignment` repository
+1. Add your GitHub repository as a remote for your local `githomework1` repository
     * `git remote add origin <url>`
-1. Push your local `gitassignment` repository to GitHub
+1. Push your local `githomework1` repository to GitHub
 1. Include your repository URL in your submission
 
+### Task 3
+1. Create a new branch called `playing-with-bash`
+2. Create 5 empty files with the names up to you
+3. Echo into the 5 empty files a few sentences about anything
+4. Stage, commit and push your work on GitHub
+5. Open up a pull request
+
 ### Submission
-* Submit your screenshots and repository URL pasted into a Google Docs file inside your submission folder
+
+1. When you are done, please commit your changes, and push it to GitHub.
+2. Open up a pull request and add your TA as a reviewer. You may need to add them to your repository.
```

Comment:
Also this.. what is the source/destination for the pull request. Since the main branch is used for Tasks 1/2, should students add the TA as a reviewer on the main <- playing-with-bash PR? Or instructor/main <- student/main?

Reply:
It will be the former. Homework is optional and this gives the students an opportunity to practice using branches to merge changes into the `main` branch.